client/src/components/Header/Header.style.js

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+
+export const H1 = styled.h1`
+  color: #fff;
+  font-size: 28px;
+  padding: 0;
+  margin: 0;
+  height: 36px;
+  img {
+    height: 56px;
+  }
+`;
+
+export const H2 = styled.h2`
+  color: #000;
+`;
```

Comment:
I can't see where you have used H2 in header or anywhere else?

Reply:
Yep was for testing. Removed now.